### PR TITLE
feat: WAProto automatic update

### PR DESCRIPTION
**Pull request by LeonardoBein**
This PR create an action to update waproto once a week.

Action created:
- update-proto.yml

The action create a PR with the changes identified on whatsapp servers.

image

### CODE REVIEW

**Comment by Xevalous** [comment]
up

File: .github/workflows/update-proto.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    - name: Update WAProto.proto
+      id: wa_proto_info
+      run: |
+        yarn --cwd proto-extract start > wa-logs.txt
+        WA_VERSION=$(cat wa-logs.txt | perl -n -e'/Current version\: (.+)/ && print $1')
+        WA_JS_URL=$(cat wa-logs.txt | perl -n -e'/Found source JS URL\: (.+)/ && print $1')
+        echo "wa_version=$WA_VERSION" >> $GITHUB_OUTPUT
+        echo "wa_js_url=$WA_JS_URL" >> $GITHUB_OUTPUT
+
+    - name: GenerateStatics
```
Comment (adiwajshing) [inline .github/workflows/update-proto.yml:32]: Can just use `yarn gen:protobuf`?

File: proto-extract/package.json [hunk 2]
```diff
@@ -8,6 +8,9 @@
   "dependencies": {
     "acorn": "^6.4.1",
     "acorn-walk": "^6.1.1",
+    "pbjs": "^0.0.14",
```
Comment (adiwajshing) [inline proto-extract/package.json:11]: don't believe we need this?

Comment (LeonardoBein) [reply proto-extract/package.json]: really not necessary.

**Pull request closed without merging**